layouts/partials/microblog_opengraph.html

```diff
@@ -0,0 +1,26 @@
+<meta name="twitter:card" content="summary">
+
+{{ with .Title }}
+  <meta name="twitter:title" content="{{ . }}">
+  <meta property="og:title" content="{{ . }}">
+{{ else }}
+  <meta name="twitter:title" content="{{ .Site.Title }}">
+  <meta property="og:title" content="{{ .Site.Title }}">
+{{ end }}
+
+{{ if .IsHome }}
+  <meta name="twitter:description" content="{{ .Site.Params.itunes_description | truncate 200 }}">
+  <meta name="og:description" content="{{ .Site.Params.itunes_description | truncate 200 }}">
+  <meta name="description" content="{{ .Site.Params.itunes_description }}">
+  <meta name="twitter:image" content="{{ .Site.Author.avatar }}">
+  <meta property="og:image" content="{{ .Site.Author.avatar }}">
+{{ else }}
+  <meta name="twitter:description" content="{{ .Summary | truncate 200 }}">
+  <meta name="og:description" content="{{ .Summary | truncate 200 }}">
+  <meta name="description" content="{{ .Summary }}">
+{{ end }}
+
+{{ range .Params.photos }}
+  <meta name="twitter:image" content="{{ . }}">
+  <meta property="og:image" content="{{ . }}">
```

Comment:
You could use `"https://micro.blog/photos/1200x/{{ . }}"` to get an image that is resized to 1200x and maintains aspect ratio for the height to address the expected dimensions.

Reply:
Ahh, I forgot Micro.blog had that built in! Good call.